web/src/settings_org.ts

```diff
@@ -84,7 +84,7 @@ export function maybe_disable_widgets(): void {
     }
 
     $(".organization-box [data-name='organization-profile']")
-        .find("input, textarea, button, select")
+        .find("input, textarea, .deactivate_realm_button, select")
```

Comment:
We can remove `.deactivate_realm_button` class from here as well once we hide the section for non-owners.